org.gnome.chess.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "org.gnome.chess",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "gnome-chess",
+    "copy-icon": true,
+    "rename-icon": "gnome-chess",
+    "rename-desktop-file": "gnome-chess.desktop",
+    "rename-appdata-file": "gnome-chess.appdata.xml",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* dconf */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        /* Documents directory access for storing saved games */
+        "--filesystem=xdg-documents"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "append-path": "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup": ["/share/gnuchess", "/share/info", "/share/man"],
+    "modules": [
+        {
+            "name": "gnuchess",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ftpmirror.gnu.org/chess/gnuchess-6.2.3.tar.gz",
+                    "sha256": "78999176b2f2b5e0325bcc69749b7b2cefb7b1ef4f02d101fa77ae24a1b31b82"
+                }
+            ]
+        },
+        {
+            "name": "gdk-pixbuf2",
+            "buildsystem": "meson",
+            "cleanup-platform": [ "/bin"],
+            "sources": [
+                 {
+                     "type": "archive",
+                     "url": "https://download.gnome.org/sources/gdk-pixbuf/2.37/gdk-pixbuf-2.37.0.tar.xz",
```

Comment:
Is there anything wrong with the versions shipped in the runtime?

Reply:
Is gdk-pixbuf really a part of the gnome runtime? I just tried to remove it and librsvg failed to build:
`libtool: install: /usr/bin/install -c .libs/libpixbufloader-svg.soT /usr/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so`
`/usr/bin/install: cannot remove '/usr/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so': Read-only file system`
`make[3]: *** [Makefile:527: install-gdk_pixbuf_moduleLTLIBRARIES] Error 1`
`make[3]: Leaving directory '/run/build/librsvg/gdk-pixbuf-loader'`
`make[2]: *** [Makefile:752: install-am] Error 2`
`make[2]: Leaving directory '/run/build/librsvg/gdk-pixbuf-loader'`
`make[1]: *** [Makefile:1725: install-recursive] Error 1`
`make[1]: Leaving directory '/run/build/librsvg'`
`make: *** [Makefile:2031: install] Error 2`

Reply:
Failed to build for different reason than missing gdk-pixbuf.

Reply:
I see only error with moving libpixbufloader-svg.so to /usr/lib. Why the libtool is doing this when gdk-pixbuf is part of the runtime?

Reply:
Yeah, I don't see a good way to make gdk-pixbuf look for loaders in additional directories. @TingPing, any suggestions?

Reply:
librsvg itself is part of the runtime. Why did you decide to bundle it? That should definitely not be required.

The upstream flatpak-builder manifest only bundles gnuchess.

Reply:
@mcatanzaro The fdo runtime doesn't contain vala, thus vala bindings for libs such as librsvg are not there. The gnome runtime needs to rebuild it with the bindings.